applications/ava-gst-deploy/README.md

```diff
@@ -0,0 +1,138 @@
+```text
+SPDX-License-Identifier: Apache-2.0
+Copyright (c) 2021 InReality LLC
+```
+<img src="https://media-exp1.licdn.com/dms/image/C4E0BAQG5Q1SR05amJg/company-logo_200_200/0/1519901343043?e=1639008000&v=beta&t=76nSvaqAh7_T8o4cwAzYvBvE6w095KSFtXo1iTQHB-M" 
+     alt="AVA-GST by InReality LLC" width="120">
+<h1>InReality DLAV</h1>
+<p>
+InReality's Deep Learning Anonymous Analytics Engine helps companies with their vision based analytical needs by providing user-friendly user inferface and superior user experience. The engine comes with pluggable best-in-class detection AI models built-in.
+</p>
+<p>
+Now DLAV is more accessible than ever by making it available through Intel's OpenNess platform. You just have to use the helm chart provided to deploy our DLAV into your own environment.
+</p>
+<h1>Techical Details</h1>
+
+## **Pre Requisites – Resources Required**
+
+| **Resource Information**           |                      |
+|------------------------------------|----------------------|
+| Application Type                   | Video-Analytics		| 
+| Compute  (vCores)                  | 8                    |  
+| Memory (RAM)                       | 16 GB                |  
+| Storage 				       | 100  GB              |  
+| Host OS                            | CentOS 7.6.1810      | 
+  
+## **Where to Purchase**
+Contact sales@inreality.com
+
+
+## Pre Requisites - Installing OpenNESS
+|**Configuration**                   |                       |
+|  ----------------------------------|-----------------------|
+| OpenNESS Version                   | 20.12.02		         |
+| Flavor Used 					     | media-analytics			 |
+| Distribution						 | IDO    	     |
+
+
+Follow below link to setup controller and edge-node for installing OpenNESS.
+
+https://github.com/open-ness/specs/blob/openness-20.12.02/doc/getting-started/network-edge/controller-edge-node-setup.md
+
+* Go to openness-experience-kits/ directory and comment out the grub role (role: machine_setup/grub) in the **network_edge.yml** file.
+* Run the deployment script as ./deploy_ne.sh 
+
+<h2>Docker Image</h2>
+<p>
+The base image is available at dockerhub <a href="https://hub.docker.com/r/irllc/ava-gst">https://hub.docker.com/r/irllc/ava-gst</a>. This is a public image made available by InReality LLC.
+</p>
+<p>
+Do take note that you would need a license from InReality LLC in order to run this image successfully. 
+You can contact us at <a href="mailto:sales@inreality.com?">Sales at InReality</a> to request for you license today.
+</p>
+
+
+## Loading Docker Images
```

Comment:
Please remove this section and rename the above Docker Image section as "Loading Docker Images"
Also add the command to do so on the node like docker load -i name of the tar image file